[Feature]  Added MongoDB controllers for all specified models - Tweet, Video, Subscription, Playlist, Like, Comment, Dashboard (not a model actually).

RashidSiddiqui2004 opened the pull request:
## Description

This pull request addresses the task of implementing MongoDB controllers for the specified models.

### Changes Made

- Implemented MongoDB controllers for the following models:
  - Tweet
  - Video
  - Subscription
  - Playlist
  - Like
  - Comment
  - Dashboard
  - HealthCheck
  - 
### Documentation

- Updated inline comments for clarity. 
 
![chaiAurCode](https://github.com/hiteshchoudhary/chai-backend/assets/124426004/66329d7b-df20-4531-bda2-2761b51f7386)

RashidSiddiqui2004 committed:
updated tweet and subsciption controllers

RashidSiddiqui2004 committed:
updated playlist, comment and tweet controllers

RashidSiddiqui2004 committed:
updated video, like, and healthcheck controllers

RashidSiddiqui2004 committed:
undone changes

lade6501 commented on src/controllers/comment.controller.js:
```diff
@@ -4,23 +4,148 @@ import {ApiError} from "../utils/ApiError.js"
+        const comment = await Comment.findById(commentId)
+        
+        // Check if the comment exists
+        if (!tweet) { 
```
It should be comment.
line no `97` `if(!tweet)` should be `if(!comment)`

RashidSiddiqui2004 replied:
yeah thank you

ThePerfectBlue committed:
Made some changes in like.controller.js

=>  getLikedVideos was also returning liked Tweets and liked Comments made by the user.

=>  A couple of changes in toggleCommentLike & toggleTweetLike

RashidSiddiqui2004 committed:
Merge pull request #1 from ThePerfectBlue/patch-1

Made some changes in like.controller.js
